test/micro/org/openjdk/bench/java/lang/foreign/BulkOps.java

```diff
@@ -175,7 +176,7 @@ public void buffer_copy() {
     @CompilerControl(CompilerControl.Mode.DONT_INLINE)
     @OutputTimeUnit(TimeUnit.NANOSECONDS)
     public void segment_copy_static_dontinline() {
-        MemorySegment.copy(bytes, 0, segment, JAVA_INT, 0, bytes.length);
+        MemorySegment.copy(bytes, 0, segment, JAVA_BYTE, 0, bytes.length);
```

Comment:
Note that `bytes` has type `int[]`, so using `JAVA_INT` was the right choice. Maybe you just want to rename the `bytes` field to `int` ?